### PR TITLE
fixed kevin's changes, reorganized javascript for simpler workflow

### PR DESCRIPTION
**Pull request by bowenbrinegar**
I think this is the right file set to merge. I used your stuff and applied it to the previous workflow, there were two function that weren't necessary, but the rest of the stuff was good. I think all together the code in this project is looking really good. Let's Google Hangout tonight to discuss the changes.

### CODE REVIEW

**Comment by kcgilland143** [comment]
I like where you're going! I was gonna suggest that we do something exactly like that. the only problem is the merge conflicts

File: assets/javascript/battle.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        ref.remove()
+        // load the you won the pokemon page
+      };
+
+      // if (catchHealth <= 0) {
+      //   load you lost the pokemon page
+      // }
+  });
+
+  $('#catchButton').unbind().click(function() {
```
Comment (kcgilland143) [inline assets/javascript/battle.js]: why unbind?

File: assets/javascript/render.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  $('#catch').empty();
+  var nameEntry = $('<h3>').text(pokeName)
+  var healthEntry = $('<h2>').text(catchHealth)
+  var imageEntry = $("<img class='pokeBattle'>").attr("src", pokeImage)
+  $("#catch").append(imageEntry, healthEntry, nameEntry)
+
+  battleMode();
+})
+
+function loadPokemon() {
```
Comment (kcgilland143) [inline assets/javascript/render.js:70]: why do you hate my render function so much :'(

**Review by kcgilland143 — COMMENTED** [review]
had to keep app.js to reference for any commits started before reorganization, everything else was pretty straight forward

**Comment by bowenbrinegar** [comment]
It still has significant problems, but I think its closer to what we need. Should I merge?

**Comment by kcgilland143** [comment]
good to go!

File: assets/javascript/render.js (hunk 2 above)
Comment (bowenbrinegar) [reply assets/javascript/render.js:70]: I'd don't hate it. I actually think it's a pretty cool idea. I just don't think it's application fits the problem at hand, however. If you can make it so its applicable to two different types of rendering (battle mode & pouch display--which need to be different at least from a standpoint of changeable health; & potentially a "you just collected X pokémon with Y stats" modal), then I say go for it. If you do, you need to take into consideration UI perspectives and the different types of displays we'll need. For instance the pouch may need different data presentation that the battle mode, and the image in the battle mode may be a different size than the image in the pouch. And Jane needs to be able to figure out how to make those changes/

I think it's easier to just render it two different ways; which will provide time for other cool features, but if you want to focus in on the render function I get it. Sometimes these things are worth figuring out.

I'm just trying to make things go smooth.

**Pull request merged**